legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -192,7 +192,21 @@ def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
         elif legal_name != response_json['business']['legalName']:
             msg.append({'error': 'Legal name does not match with company legal name from Colin.',
                         'path': business_legal_name_path})
-        elif founding_date != response_json['business']['foundingDate']:
-            msg.append({'error': 'Founding date does not match with founding date from Colin.',
-                        'path': business_founding_date_path})
+        else:
+            # Check if founding date is None
+            if founding_date is None:
+                msg.append({
+                    'error': 'Founding date is missing in the filing data.',
+                    'path': business_founding_date_path
+                })
```

Comment:
Do we need this?

Reply:
If the schema already makes this required then no.

Reply:
its not a required filed as per schema

Reply:
Just to confirm how to handle this: if per filing json is none, do we check Colin returns none too (does this happen?); or return error message?